Description: Remove simplejson usage
 It serves no purpose and it fails the JSON tests included
 in Raven when present.
Author: Gilles Dubuc <gilles@wikimedia.org>

--- raven-5.18.0.orig/raven/utils/json.py
+++ raven-5.18.0/raven/utils/json.py
@@ -11,10 +11,7 @@ from __future__ import absolute_import
 import codecs
 import datetime
 import uuid
-try:
-    import simplejson as json
-except ImportError:
-    import json
+import json
 
 try:
     JSONDecodeError = json.JSONDecodeError
